flake.lock

```diff
@@ -36,11 +36,11 @@
     },
     "nixpkgs": {
       "locked": {
-        "lastModified": 1683020768,
```

Comment:
I updated the flake lockfile to ensure that Cargo version is >= 1.68.

Reply:
We also test against the latest NixOS release (new release comes out at the end of the month). Also worth noting that sparse registires are getting fully stabilized in 1.70 which also comes out in two weeks